command/v7/labels_command.go

```diff
@@ -10,15 +10,23 @@ import (
 	"code.cloudfoundry.org/cli/command/v7/shared"
 	"code.cloudfoundry.org/cli/types"
 	"code.cloudfoundry.org/cli/util/ui"
+	"fmt"
 )
 
+//go:generate counterfeiter . GetLabelActor
+
+type GetLabelActor interface {
```

Comment:
The convention we are trying to follow is that the Actor for `FooCommand` is `FooActor`. We plan to merge this and make the change ourselves to save the additional back-and-forth. 

See [`command/v7/domains.go`](https://github.com/cloudfoundry/cli/blob/92ecc3a56b6971696d84ad908f204b62ed8b765a/command/v7/domains_command.go) for an example of this.

@sannidhi , you  explained that you and @ericpromislow  tried to fix the fakes by adding this "GetLabelActor" last week in Seattle. Sorry I couldn't have just given you this feedback then. Was there another convention that you had seen elsewhere?

Reply:
@bwasmith  - Sorry just saw this. We saw there was a SetLabelActor and a DeleteLabelActor so we went with ActionActor (where get was the action in our case).